lib/byron/bench/Restore.hs

```diff
@@ -190,16 +187,16 @@ main = do
     let args =
             [ "run"
             , "--database-path", nodeDB
-            , "--topology", configs </> "mainnet-topology.json"
+            , "--topology", topology
             , "--socket-path", socketPath
-            , "--config", configs </> "configuration-mainnet.yaml"
+            , "--config", config
             , "--port", "7776"
             ]
-    -- NOTE: It is warned against using @NoStream@ to supress output on POSIX
-    -- systems, since it could lead to the process throwing errors.
-    -- It appears to work fine stopping the live view of cardano-node,
-    -- however.
-    let cmd = Command "cardano-node" args (return ()) NoStream NoStream
+    let cmd = Command "cardano-node" args (return ()) Inherit Inherit
```

Comment:
There is actually a `Cardano.Wallet.Byron.Launch.withCardanoNode` function.
Would this also be suitable for the restore benchmark?

Reply:
That module currently only contains self-node/integration-style setup… Maybe the module should be re-named if we don't end up adding any "normal" launch code